### PR TITLE
Add trino component into hadoop-testing

### LINKED ISSUE #11: Add presto/trino component into hadoop-testing
As we all know, presto and trino both are popular AD-Hoc engines for the data warehouse and lakehouse in the Hadoop ecosystem. It's valuable to add them to the playground.

### PR DESCRIPTION
**Pull request by yanghua**
Close #11

### CODE REVIEW

**Comment by yanghua** [comment]
It's a draft.

File: hadoop-master1/files/etc/trino/conf/config.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+coordinator=true
+node-scheduler.include-coordinator=true
```
Comment (pan3793) [inline hadoop-master1/files/etc/trino/conf/config.properties:2]: typically should be `false` in production deployment?

File: hadoop-master1/files/etc/trino/conf/trino-env.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+export JAVA_HOME=/opt/openjdk-21
```
Comment (pan3793) [inline hadoop-master1/files/etc/trino/conf/trino-env.sh:1]: emm... this file seems does not take effect

File: hadoop-master1/files/etc/trino/conf/config.properties (hunk 1 above)
Comment (yanghua) [reply hadoop-master1/files/etc/trino/conf/config.properties:2]: Yes, currently single-node multiple roles that only for testing.

File: hadoop-master1/files/etc/trino/conf/trino-env.sh (hunk 2 above)
Comment (yanghua) [reply hadoop-master1/files/etc/trino/conf/trino-env.sh:1]: Yes, old test file, will remove it.

**Comment by yanghua** [comment]
image

@pan3793 It's ready for review.

File: hadoop-master1/files/etc/trino/conf/node.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+node.environment=production
+node.id=trino-coordinator
```
Comment (pan3793) [inline hadoop-master1/files/etc/trino/conf/node.properties:2]: let's use the hostname
```suggestion
node.id=hadoop-master1
```

File: hadoop-worker3/files/etc/trino/conf/log.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+io.trino=INFO
+
```
Comment (pan3793) [inline hadoop-worker3/files/etc/trino/conf/log.properties:2]: remove blank line

**Review by pan3793 — APPROVED** [review]
almost good, thanks

File: README.md [hunk 5]
```diff
@@ -77,6 +78,7 @@ Once the testing environment is fully operational, the following services will b
 - Hadoop YARN: http://hadoop-master1.orb.local:8088
 - Hadoop MapReduce JobHistory: http://hadoop-master1.orb.local:19888
 - Ranger Admin: http://hadoop-master1.orb.local:6080 (admin/Ranger@admin123)
+- Trino Web UI: http://hadoop-master1.orb.local:18081 (admin/)
```
Comment (pan3793) [inline README.md:81]: I remember there is no username/password by default
```suggestion
- Trino Web UI: http://hadoop-master1.orb.local:18081
```

Comment (yanghua) [reply README.md:81]: Currently, it needs a user but no password, if there is no additional configuration.

**Comment by pan3793** [comment]
Thanks, merged.

File: README.md (hunk 5 above)
Comment (yanghua) [reply README.md:81]: The description is here: https://trino.io/docs/current/admin/web-interface.html#authentication

**Pull request merged**